### ISSUE TITLE
Variable splitting in multiple History collections

### ISSUE THREAD

**Issue by atrayano** [GEOS-ESM/MAPL#673]
This is a bug, that can be reproduced when we try to split the same variable in more that 1 collection. The issue is that result of the splitting is a new field, and currently we put it in the same export state as the original field. This works as long as the same variable is not split again in another collection. The second time we do this, ESMF will return an error that the export state already contains the split field

**Comment by mathomp4** [comment]
@atrayano I believe this is fixed, yes?

**Comment by atrayano** [comment]
@mathomp4 this is definitely fixed but I am not 100% if it got merged into develop. I will check and get back to you.

**Comment by atrayano** [comment]
@mathomp4 I just checked. It is not merged to develop. I have two branches that I passed to Elliot for testing (and 1 of them addresses #673 and 1 for #674). But Elliot needed one more thing from me to fully test (overload for MAPL_ConfigSetAttribute for array of integers and reals). This is all done, but Tom suggested a freeze MAPL until we deliver a stable version to NOAA, so I am putting my PR's on hold until then
